app/api/catalogues/[catalogueId]/quiz/submit/route.ts

```
@@ -0,0 +1,74 @@
+        catalogueId: catalogueId,
+      },
+    });
+
+    const response = calculateScore(
+      catalogueId,
+      correctAnswers,
+      validatedResult.data
+    );
+
+    return NextResponse.json(response);
+  } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)